library/hkdf.c

```diff
@@ -62,6 +62,10 @@ int mbedtls_hkdf_extract( const mbedtls_md_info_t *md,
     {
         size_t hash_len;
 
+        if( salt_len != 0 ) {
```

Comment:
I hate to be the one to point out a silly style issue, but the bracing doesn't follow our [C coding standard](https://tls.mbed.org/kb/development/mbedtls-coding-standards). Sorry!

Reply:
@sbutcher-arm fixed